index.html

```diff
@@ -15,29 +17,31 @@
     <i class='bx bx-menu bx-sm'></i>
   </nav>
   <section class="headline">
+    <div class="nav">
+      <ul><li>Portoflio</li>
+        <li>About</li>
+        <li>contact</li>
+      </ul>
+    </div>
     <div class="container">
-      <h1>Hey there. <br>
-        I’m Tsige.</h1>
+      <h1>Hey there.I’m Tsige.</h1>
       <h2 class="green">I’m a software developer! </h2>
     </div>
     <p>I can help you build any software as your need
       including DesktopApp ,Mobile feature or website.
       For more Please Look through
       some of my work and experience that I have!</p><br>
-    <div class="contact"><ul>
-      <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
-      <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img src="icons/Linkedin icon.svg"></i></a></li>
-      <li><a href=""><img src="icons/medium.svg"></i></a></li>
-      <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
-      <li><a href=""><img src="icons/angellist.svg"></a></li>
+    <div class="contact1"><ul>
+      <li><a href="https://github.com/tsigewossne" class="fa fa-github"></a></li>
+      <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/" class="fa fa-linkedin"></a></li>
+      <li><a href="#" class="fa fa-angellist"></a></li>
+      <li><a href="https://twitter.com/TsigeWo" class="fa fa-twitter"></a></li>
+      <li><a href="#" class="fa fa-medium"></a></li>
     </ul></div>
```

Comment:
Great job on addressing the previous reviewer's requests and making improvements to your project. However, there is still an issue with the social icons in the headline section not displaying properly on different devices.

Please carefully review the design requirements and ensure that the social icons are correctly implemented and visible on all devices. Consider using responsive techniques such as media queries to adjust the placement and visibility of the icons based on the screen size.

By following the design guidelines closely and making the necessary updates to your code, you will ensure that the social icons are consistently displayed across different devices. Keep up the good work! 👍